code/game/objects/items/mop.dm

```diff
@@ -107,8 +113,11 @@
 /obj/item/mop/cyborg
 	insertable = FALSE
 
+/obj/item/mop/cyborg/attack_self(mob/user)
+	return
+
 /obj/item/mop/advanced
-	desc = "The most advanced tool in a custodian's arsenal, complete with a condenser for self-wetting! Just think of all the viscera you will clean up with this!"
+	desc = "The most advanced tool in a custodian's arsenal, complete with a condenser for self-wetting! Just think of all the viscera you will clean up with this! Alt-click to toggle the condenser."
```

Comment:
You removed the altclicking functionality for some reason.

Reply:
I didn't? Are you confusing this with overwriting the bracing proc (attack_self) for cyborg subtypes?